third_party/prism.BUILD

```diff
@@ -0,0 +1,83 @@
+GENERATED_SRCS = [
+  "src/node.c",
+  "src/prettyprint.c",
+  "src/serialize.c",
+  "src/token_type.c",
+]
+
+GENERATED_HDRS = [
+  "include/prism/ast.h",
+  "include/prism/diagnostic.h",
+]
+
+genrule(
+  name = "generate_templates",
+  srcs = ["templates/template.rb"],
+  outs = GENERATED_HDRS + GENERATED_SRCS,
+  cmd = """
+    # This is a workaround; without guidance, Bazel will try to use the system Ruby,
+    # which is too old to install gems and run this script.
+    #
+    # Pass the RUBY_PATH variable to the build using the --define flag, e.g.
+    # ./bazel build //main:sorbet --config=dbg --define RUBY_PATH=/path/to/ruby
+    export PATH="$(RUBY_PATH):$$PATH"
```

Comment:
Like the comment says -- Ideally, Bazel would use a new version of Ruby that's built along with this project, but I'm having trouble running Sorbet's Ruby builds. This is not a blocking issue for a prototype (I'm making the assumption that I could get Bazel to both build Ruby and use that Ruby to execute this script), so I'm planning to address it once we advance past the prototype stage.

In the meantime, you can make sure Bazel uses the correct version of Ruby by passing the Ruby path to the build command, like so:

`./bazel build //main:sorbet --config=dbg --define RUBY_PATH=/path/to/ruby`